homeassistant/components/hunterdouglas_powerview/manifest.json

```diff
@@ -18,6 +18,6 @@
   },
   "iot_class": "local_polling",
   "loggers": ["aiopvapi"],
-  "requirements": ["aiopvapi==2.0.4"],
-  "zeroconf": ["_powerview._tcp.local."]
+  "requirements": ["aiopvapi==3.0.1"],
+  "zeroconf": ["_powerview._tcp.local.", "powerview-g3.local."]
```

Comment:
```suggestion
  "zeroconf": ["_powerview._tcp.local.", "powerview-g3._tcp.local."]
```

Reply:
This has to have `_tcp` or `_udp` in it according to https://www.rfc-editor.org/rfc/rfc6763.html

Reply:
I don't believe that is the case for the primary - it was discussed on a thread here previously (on my phone would have to search for it)

But I believe the _tcp reports all hubs, where the non _tcp only contains the primary hub